modules/claims_api/config/schemas/v2/request_bodies/disability_compensation/example.json

```diff
@@ -6,55 +6,52 @@
       "veteranIdentification": {
-          "mailingAddress": {
+        "currentlyVaEmployee": false,
+        "mailingAddress": {
             "numberAndStreet": "1234 Couch Street",
-            "apartmentOrUnitNumber": "Apt. 22",
+            "apartmentOrUnitNumber": "22",
             "city": "Portland",
             "country": "USA",
-            "zipFirstFive": "12345",
-            "zipLastFour": "6789",
+            "zipFirstFive": "41726",
+            "zipLastFour": "1234",
             "state": "OR"
-          },
-          "currentlyVAEmployee": false,
-          "serviceNumber": "123456789",
-          "emailAddress": {
-              "email": "claimant@example.com",
-              "agreeToEmailRelatedToClaim": true
-          },
-          "veteranNumber": {
-              "telephone": "555-5555",
-              "internationalTelephone": "555-5555"
-          },
-          "vaFileNumber": "12345678"
-      },
-      "changeOfAddress": {
-          "country": "USA",
-          "type": "INTERNATIONAL",
-          "addressLine1": "1234 Couch Street",
-          "dates": {
-            "beginningDate": "2018-11-31",
-            "endingDate": "2019-07-11"
-          },
-          "typeOfAddressChange": "Permanent",
-          "numberAndStreet": "1234 Couch Street",
-          "apartmentOrUnitNumber": "Apt. 22",
-          "city": "Portland",
-          "zipFirstFive": "12345",
-          "zipLastFour": "1234",
-          "state": "OR"
-      },
+        },
+        "serviceNumber": "123456789",
+        "emailAddress": {
+            "email": "valid@somedomain.com",
+            "agreeToEmailRelatedToClaim": true
+        },
+        "veteranNumber": {
+            "telephone": "5555555555",
+            "internationalTelephone": "5555555555"
+        },
+        "vaFileNumber": "AB123CDEF"
+    },
+    "changeOfAddress": {
+      "dates": {
+          "beginningDate": "2012-11-31",
+          "endingDate": "2013-10-11"
+        },
+        "typeOfAddressChange": "TEMPORARY",
+        "numberAndStreet": "10 Peach St",
+        "apartmentOrUnitNumber": "22",
+        "city": "Atlanta",
+        "zipFirstFive": "42220",
+        "zipLastFour": "9897",
+        "state": "GA",
+        "country": "USA"
+  },
       "homeless": {
-        "pointOfContact": "Jane Doe",
-        "pointOfContactNumber": "123-456-7890",
-        "currentlyHomeless": {
-          "homelessSituationOptions": "FLEEING_CURRENT_RESIDENCE",
-          "otherDescription": ""
+        "pointOfContact": "john stewart",
+        "pointOfContactNumber": {
+            "telephone": "5555555555",
+            "internationalTelephone": "5555555555"
         },
-        "riskOfBecomingHomeless": {
-          "livingSituationOptions": "other living situation",
-          "otherDescription": ""
+        "currentlyHomeless": {
+            "homelessSituationOptions": "FLEEING_CURRENT_RESIDENCE",
+            "otherDescription": "ABCDEFGHIJKLM"
         }
```

Comment:
It looks like the "riskOfBecomingHomeless" object got removed, or changed to "currentlyHomeless"?  It seemed "riskOfBecomingHomeless" and then inside that, "livingSituationOptions" still needed to be there?

Reply:
I'm still not seeing "riskOfBeingHomeless" in the "example" in the swagger docs, but it needs to be there right?
<img width="583" alt="Screenshot 2023-06-05 at 9 09 44 AM" src="https://github.com/department-of-veterans-affairs/vets-api/assets/129893414/2d7348a0-eeac-4b4b-8444-3663c948a3d6">

Reply:
what do you think?